conf/response/submit/222290b4-06e3-4fef-a555-6fd0877dc7ca-response.json

```diff
@@ -93,7 +93,7 @@
         {
           "metadata": [
             {
-              "TYPEID": "string"
+              "typeID": "string"
```

Comment:
Hi Sam, could you please revisit this whole casing this, because on 07-feb the case fix which was commited was changing it from "typeIDs" to "typeId" -- note d is in lower case, but now its upper case. What is the actual agreed value?